test/system/archive_baseline.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/sh -f
+
+echo
+
+if [ $# -ne 1 ]; then
+    echo "Invoke archive_baseline.sh -help for usage."
+    exit 1
+fi
+
+if [ $1 == "-help" ] || [ $1 == "--help" ]; then
+cat << EOF1
+NAME
+
+	archive_baseline.sh - archive pretag baselines to set locations on
+                              izumi and derecho.
+
+
+SYNOPSIS
+
+	archive_baseline.sh TAGNAME
+	  [-help]
+
+
+ENVIROMENT VARIABLES
+
+	CESM_TESTDIR - Directory that contains the CESM finished results you wish to archive.
+	CAM_FC      - Compiler used, only used on izumi (GNU,NAG), where the compiler
+                      name is appended to the archive directory.
+
+
+BASELINE ARCHIVED LOCATION
+
+	izumi:     /fs/cgd/csm/models/atm/cam/pretag_bl/TAGNAME_gnu
+	                   /fs/cgd/csm/models/atm/cam/pretag_bl/TAGNAME_nag
+        derecho:  /glade/campaign/cesm/community/amwg/cam_baselines/TAGNAME
+
+
+
+HOW TO USE ARCHIVE BASELINES
+
+	Set BL_TESTDIR to the archived baseline you wish to load.
+
+
+WORK FLOW
+
+	This is an example for izumi.
+
+	Modify your sandbox with the changes you want.
+        setenv CAM_FC GNU
+        setenv CAM_TESTDIR /scratch/cluster/fischer/cam5_2_06
+        Run the cam test suite.
+        Make your trunk tag
+	archive_baseline.sh cam5_2_06
+
+	Create a new sandbox.
+        setenv CAM_FC GNU
+	setenv CAM_TESTDIR /scratch/cluster/fischer/cam5_2_07
+        setenv BL_TESTDIR /fs/cgd/csm/models/atm/cam/pretag_bl/cam5_2_06_gnu
+        Run the cam test suite.
+        Make your trunk tag
+        archive_baseline.sh cam5_2_07
+
+
+WARNING
+
+	System changes can cause answer changes. So you may need to create new baselines
+        if you are getting unexpected baseline failures.
+
+EOF1
+exit
+fi
+
+hostname=`hostname`
+case $hostname in
+
+  iz*)
+    echo "server: izumi"
+    if [ -z "$CAM_FC" ]; then
+      CAM_FC="GNU"
+    fi
+    test_file_list="tests_pretag_izumi_${CAM_FC,,}"
+    cam_tag=$1_${CAM_FC,,}
+    baselinedir="/fs/cgd/csm/models/atm/cam/pretag_bl/$cam_tag"
+  ;;
+
+  de*)
+    echo "server: derecho"
+    if [ -z "$CAM_FC" ]; then
+      CAM_FC="INTEL"
+    fi
+    test_file_list="tests_pretag_derecho"
+    cam_tag=$1
+    baselinedir="/glade/campaign/cesm/community/amwg/cam_baselines/$cam_tag"
```

Comment:
I think we want to change it to `sima_baselines` here:

```suggestion
    baselinedir="/glade/campaign/cesm/community/amwg/sima_baselines/$cam_tag"
```

Reply:
done